# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/zfs-fuse-uclibc.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- src/SConstruct.vanilla	2007-01-16 09:41:12.000000000 +0000
+++ src/SConstruct	2007-01-16 09:41:24.000000000 +0000
@@ -10,8 +10,8 @@
 env = Environment()
 
 env['CC'] = 'gcc'
-env['LINKFLAGS'] = Split('-pipe -Wall -Werror')
-env['CCFLAGS'] = Split('-pipe -Wall -Werror -std=c99 -Wno-missing-braces -Wno-parentheses -Wno-uninitialized -fno-strict-aliasing -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -D_REENTRANT -DTEXT_DOMAIN=\\"zfs-fuse\\"')
+env['LINKFLAGS'] = Split('-pipe -Wall')
+env['CCFLAGS'] = Split('-pipe -Wall -std=c99 -Wno-missing-braces -Wno-parentheses -Wno-uninitialized -fno-strict-aliasing -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -D_REENTRANT -DTEXT_DOMAIN=\\"zfs-fuse\\"')
 
 debug = int(ARGUMENTS.get('debug', '1'))
 
--- src/lib/libnvpair/nvpair.c.vanilla	2007-01-16 09:40:13.000000000 +0000
+++ src/lib/libnvpair/nvpair.c	2007-01-16 09:40:16.000000000 +0000
@@ -26,7 +26,7 @@
 
 
 
-#include <sys/stropts.h>
+//#include <sys/stropts.h>
 #include <sys/debug.h>
 #include <sys/isa_defs.h>
 #include <sys/int_limits.h>
--- src/lib/libnvpair/nvpair_alloc_fixed.c.vanilla	2007-01-16 09:40:47.000000000 +0000
+++ src/lib/libnvpair/nvpair_alloc_fixed.c	2007-01-16 09:40:50.000000000 +0000
@@ -26,7 +26,7 @@
 
 
 
-#include <sys/stropts.h>
+//#include <sys/stropts.h>
 #include <sys/isa_defs.h>
 #include <sys/nvpair.h>
 #include <sys/sysmacros.h>
--- src/SConstruct.vanilla	2007-01-18 19:23:17.000000000 +0000
+++ src/SConstruct	2007-01-18 19:23:34.000000000 +0000
@@ -11,7 +11,7 @@
 
 env['CC'] = 'gcc'
 env['LINKFLAGS'] = Split('-pipe -Wall')
-env['CCFLAGS'] = Split('-pipe -Wall -std=c99 -Wno-missing-braces -Wno-parentheses -Wno-uninitialized -fno-strict-aliasing -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -D_REENTRANT -DTEXT_DOMAIN=\\"zfs-fuse\\"')
+env['CCFLAGS'] = Split('-pipe -Wall -std=c99 -Ilibintl -Wno-missing-braces -Wno-parentheses -Wno-uninitialized -fno-strict-aliasing -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -D_REENTRANT -DTEXT_DOMAIN=\\"zfs-fuse\\"')
 
 debug = int(ARGUMENTS.get('debug', '1'))
 
--- /dev/null
+++ src/libintl/libintl.h	2007-01-18 19:30:36.000000000 +0000
@@ -0,0 +1,4 @@
+#define gettext(str) str
+#define dgettext(dom,str) str
+#define textdomain(arg) 0
+#warning here
